packages/react/src/patterns/blocks/LogoGrid/LogoGrid.js

```diff
@@ -18,61 +19,68 @@ const { prefix } = settings;
  * Logo Grid
  *
  * @param {object} props Props object
- * @param {string} props.title Title for the Logo Grid Pattern
- * @param {string} props.theme theme name
+ * @param {string} props.heading Heading for the Logo Grid Pattern
  * @param {Array} props.logosGroup Array of object with label, imgSrc and altText properties
+ * @param {object} props.cta CTA object
+ * @param {boolean} props.hideBorder Set to true to hide the grid bottom border
  * @returns {*} Logo Grid Pattern JSX object
  */
-const LogoGrid = ({ theme, title, logosGroup }) => {
+const LogoGrid = props => {
```

Comment:
@reesedownes why was this converted to just `props`? We should probably have as separate props so we can stay consistent.

Reply:
oh. i actually thought "props" was the way we were supposed to do it. i'll revert.